Framework/Geometry/src/Objects/MeshObject2D.cpp

```diff
@@ -6,6 +6,7 @@
 // SPDX - License - Identifier: GPL - 3.0 +
 #include "MantidGeometry/Objects/MeshObject2D.h"
 #include "MantidGeometry/Objects/IObject.h"
+#include "MantidGeometry/Objects/MeshObjectCommon.h"
```

Comment:
Includes are out of order here

Reply:
I agree here.

Reply:
It's correct here as well then.

Reply:
clang-format should sort out the include order these days. It has an exception that keeps the header for the current implementation file at the top.